react/components/AddressCompletionForm.tsx

```diff
@@ -42,25 +45,29 @@ const AddressCompletionForm: React.FC<Props> = ({
   const {
     orderForm: { clientProfileData, canEditData },
   } = useOrderForm()
+  const { searchedAddress } = useOrderShipping()
 
-  const { address, invalidFields } = useAddressContext()
+  const { address, invalidFields, rules } = useAddressContext()
   const [buyerIsReceiver, setBuyerIsReceiver] = useState(
     !address.isDisposable || canEditData
   )
 
+  const { invalidFields: mandatoryEditableFields } = validateAddress(
+    searchedAddress,
+    rules
+  )
+
   const { firstName, lastName } = clientProfileData!
 
   const selectedDeliveryOption = deliveryOptions.find(
     ({ isSelected }) => isSelected
   )
 
-  const handleBuyerIsReceiverChange: React.ChangeEventHandler<HTMLInputElement> = (
-    evt
-  ) => {
```

Comment:
I don't remember whether or not our eslint config was adding these parenthesis or if it is a local config of mine. can you try updating the dependencies on the root folder?

Reply:
My bad, my local `node_modules` were outdated. Didn't realize because the lint config is in the root folder, not `/react` =P

Reply:
oh, no problem!